### DISCUSSION TITLE
Where is the commit hash displayed in the GitHub iOS app?

### DISCUSSION THREAD

**Discussion by KOConchobhair** [community/community#18660] [Mobile]
One of my main reasons to use the mobile app is go to a repository's list of commits to review the changes and cross reference it with a conversation that may be going on in Slack. All from my mobile. The only way to do that is with the commit hash. It's not displayed anywhere? Yet on the details tab for a particular commit, it shows the parent commit? Why show the parent commit but not the actual commit anywhere?

Can you pretty please show the actual commit hash along with the commit message, etc. in the iOS mobile app? 
Or have I just missed it? I looked everywhere I can think of. 

Thanks!

**Comment by eliperkins** [comment]
Hey @KOConchobhair! Thanks for this feedback. You're not missing anything; not being able to see the hash of a given commit within the commit details UI is definitely an oversight.

I've jotted down your notes about this in our internal tracking issues to get this important bit of commit data represented in the UI. I'll keep you posted on when this improvement gets shipped in GitHub Mobile!
